conf/broker.conf

```diff
@@ -1261,10 +1261,13 @@ replicatorPrefix=pulsar.repl
 # due to missing ZooKeeper watch (disable with value 0)
 replicationPolicyCheckDurationSeconds=600
 
-# Default message retention time
+# Default message retention time.
+# The default value is 0, which means the data is removed after all the subscriptions are consumed.
+# Value less than 0 means messages never expire.
 defaultRetentionTimeInMinutes=0
 
-# Default retention size
+# Default retention size, less than 0 means no infinite size quota.
+# Default is 0, which means the data will be removed after all the subscriptions are consumed.
```

Comment:
<img width="1078" alt="image" src="https://user-images.githubusercontent.com/50226895/180694703-5fab2480-2205-4a7f-b1e0-4b260185acac.png">

Similar issues still exist.

Reply:
Got it. Thanks